src/reducers/books.js

```diff
@@ -1,34 +1,30 @@
-import { REMOVE_BOOK, CREATE_BOOK } from '../actions/index';
```

Comment:
- [ ] It would be better if we create an entry point for the application. Without the entry point `src/index.js` the `npm start` command fails, therefore I could not review the UI design. Kindly bring back the entry point file